examples/specs/scatter_opacity_yfield.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
I see.

Let's pick one of these `scatter_opacity` and remove `scatter_opacity` because our unit test is covering this anyway.  

My rationale: the purpose of visual test is so that we can skim the gallery and check if everything is still in good shape.  If we add too many redundant examples, there would be too many things to look at anyway. 

That said, please search through all existing examples specs.  If we use `config.mark.opacity`, let's use `encoding.opacity.value` instead so it will cover the same test coverage.